templates/project/.github/workflows/symfony-lint.yaml.twig

```diff
@@ -0,0 +1,116 @@
+# DO NOT EDIT THIS FILE!
```

Comment:
If I understand correctly, this file need to be added only if there is a test kernel.

Reply:
Yes, I have to make the needed changes.

Reply:
Fixed.